method-proposals/PROPOSAL-did-webvh.md

```diff
@@ -17,15 +22,28 @@ method that complement `did:web` include:
   embedded in the DID, is derived from the initial DID log entry. It ensures the integrity
   of the DID's history mitigating the risk of attackers creating a new object with
   the same identifier.
-- **Authorized Keys**: DIDDoc updates contain a proof signed by the DID Controllers *authorized* to
+- **Authorized Keys**: DIDDoc updates include a proof signed by the DID Controllers *authorized* to
   update the DID.
+- **Cryptographic Agility** enabled through versioned specification upgrades,
+  algorithm-identifying formats (e.g.,
+  [multihash](https://identity.foundation/didwebvh/v1.0/#term:multihash) and
+  [Data
+  Integrity](https://identity.foundation/didwebvh/v1.0/#term:data-integrity)
+  proofs), and per-entry method parameters in the DID log that enable DIDs to
+  evolve cryptographically over time. No constraints on the keys published in the DIDDoc.
 - **Pre-rotation Keys** (optional): The mechanism for publishing pre-rotation keys prevents the loss of
   control of a DID in cases where an active private key is compromised.
 - **Witnesses** (optional): The mechanism for having witnesses enables the collaborative
   approval of updates to the DID by a DID Controller before publication.
+- **Watchers** An optional mechanism for publishing the location of `did:webvh`
+  watchers in the DID log that resolvers can use as another source DID data for
+  long term resolution and detection of malicious DID Controllers.
 - **DID Portability** (optional): The mechanism for enabling portability allows
```

Comment:
Best to rewrite these to all use "streaming text", or all use "colon definition". Mixing these results in confusing text.

Reply:
Updated to use colon definition throughout